workbench-core/example/infrastructure/integration-tests/support/resources/dynamicAuthorization/identityPermission.ts

```diff
@@ -0,0 +1,27 @@
+import { IdentityPermission as Permission } from '@aws/workbench-core-authorization';
+import ClientSession from '../../clientSession';
+import { DynamicAuthorizationHelper } from '../../complex/dynamicAuthorizationHelper';
+import Resource from '../base/resource';
+
+export default class IdentityPermission extends Resource {
+  private _identityPermission: Permission;
+  public constructor(
+    identityPermission: Permission,
+    clientSession: ClientSession,
+    parentApi: string,
+    id: string
+  ) {
+    super(clientSession, 'identityPermission', id, parentApi);
+    this._api = 'identitypermission';
```

Comment:
Is not including the permission id in the api route intended?

Reply:
There is no permission id

Reply:
if thats the case, I'd recommend removing it entirely.